cpu/ppc64_cpu_test.py

```diff
@@ -147,6 +152,14 @@ def dscr(self):
         command2 = "cat /sys/devices/system/cpu/dscr_default"
         self.equality_check("DSCR", command1, command2)
 
+    def smt_loop(self):
+        """
+        Tests smt on/off in a loop
+        """
+        for _ in range(1, 100):
+            process.run("ppc64_cpu --smt=off && ppc64_cpu --smt=on",
+                        shell=True)
```

Comment:
if it is ok to test catch failure one way to check return status 

can be like :
```
for _ in range(1, 100):
    if process.system("ppc64_cpu --smt=off && ppc64_cpu --smt=on", shell=True, ignore_status=True):
         self.fail("msg")
```